src/components/Buttonmui.js

```diff
@@ -0,0 +1,18 @@
+import {Button as ButtonMUI} from "@mui/material"
+
+export const Buttonmui=({title})=>{
```

Comment:
since there is only 1 button in your implementation, Button is enough. So change the name `Buttonmiu` -> `Button`